internal/dev_server/ui/src/App.tsx

```diff
@@ -17,6 +17,9 @@ import './App.css';
 import { useEffect, useState } from 'react';
 import { Icon } from '@launchpad-ui/icons';
 
```

Comment:
We use paths like `/api/dev/projects/:projectKey` in local dev, because the `/api` prefix triggers proxy-matching behavior in the frontend devserver to forward the requests to the API service.

We want paths like `/dev/projects/:projectKey` in the published CLI, because the API server is also serving the frontend. Toggle this behavior using meta data from the Vite build.